fmfexporter/adapters/polarion/fmf_adapter_polarion.py

```diff
@@ -38,7 +39,7 @@ def get_args_parser() -> FMFAdapterArgParser:
     def convert_from(self, fmf_testcase: FMFTestCase):
         return PolarionTestCase.from_fmf_testcase(fmf_testcase)
 
-    def submit_testcase(self, fmf_testcase: FMFTestCase):
+    def submit_testcase(self, fmf_testcase: FMFTestCase, parse_response=False):
```

Comment:
@michalxo I believe you need to add some config to the polarion adapter to change this behavior.

Reply:
Actually, this can be safely removed, as this is used via 
`populate_jira(self, submitted_testcases: list):
        # Linking Test Case Work items in jira
        if PolarionArgParser.JIRA_CONFIG is not None:`

Reply:
Actually, it's done through 
```
fmfexporter/adapters/polarion/fmf_adapter_polarion.py:51

tc = self._reporter.submit_testcase(ptc, PolarionArgParser.POPUL_TC)
...
submitted_tc.extend(self._reporter.submit_testcases(polarion_test_cases, PolarionArgParser.POPUL_TC))
```
So this should be good.